src/pages/GenerateKeys/Option1.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Heading } from '../../components/Heading';
+import { Text } from '../../components/Text';
+import { Link } from '../../components/Link';
+import { Alert } from '../../components/Alert';
+import { CHAIN_NAME, IS_MAINNET } from '../../utils/envVars';
+import { Paper } from '../../components/Paper';
+import { colors } from '../../styles/styledComponentsTheme';
+
+export const Option1 = ({
+  validatorCount,
+  os,
+}: {
+  validatorCount: number | string;
+  os: string;
+}) => {
+  return (
+    <Paper className="mt20">
+      <Heading level={2} size="small" color="blueMedium" className="mb20">
+        Option 1: Use binary executable file
+      </Heading>
+      <Text weight={500}>Download the binary executable file</Text>
+      <Text>
+        Please find the binary execution file of the latest deposit-cli release
+        on{' '}
+        <Link
+          primary
+          external
+          to="https://github.com/ethereum/eth2.0-deposit-cli/releases/"
```

Comment:
It might be unclear for users where the assets are to download on this Github Releases page. We may want to specify where the assets are located and which they should download for their OS. Alternatively, we could place the link to trigger the download upon click for their OS. If we go this route we would want to make sure the user is not being phished as they could be downloading a file from an untrusted source